src/render_gl.h

```diff
@@ -88,6 +89,29 @@ enum gl_lib_type
   GL_LIB_PROGRAMMABLE,
 };
 
+#if PLATFORM_BYTE_ORDER == PLATFORM_BIG_ENDIAN
+/**
+ * The GL renderers use GL_UNSIGNED_BYTE for 32-bit texture buffers for GLES 2
+ * compatibility. This leads to some problems when packing bytes on big endian
+ * machines because each of these packings assumes GL_UNSIGNED_INT_8_8_8_8_REV
+ * (which isn't in GLES 2).
+ */
+static inline Uint32 gl_pack_u32(Uint32 x)
```

Comment:
Wouldn't it be better to use __builtin_bswap32 here?

Reply:
`__builtin_bswap32` appears to be GCC/clang only and from (this compatibility code for it)[https://gist.github.com/jtbr/7a43e6281e6cca353b33ee501421860c#file-endianness-h-L66] appears to have been introduced in GCC 4.8. MSVC has a different equivalent builtin. I'm not really against having compatibility defines for these somewhere but something like what I linked definitely doesn't belong in render_gl.h (platform_endian.h would be better). I'd rather just not for this PR.

Reply:
Added a note about byte swapping for when/if that ever becomes a thing. I'm going to mark this resolved..